### PR TITLE
Improve project

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Improve project
| Type?         | improvement
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | N/A
| How to test?  | Github Actions are all :green_circle: : https://github.com/Progi1984/blockreassurance/actions

### CODE REVIEW

**Review by NeOMakinG — APPROVED** [review]

**Review by matthieu-rolland — APPROVED** [review]

**Comment by matthieu-rolland** [comment]
@Progi1984 nice PR ! there's just a conflict to be solved

File: controllers/admin/AdminBlockListingController.php [hunk 1]
```diff
@@ -165,9 +168,9 @@ public function displayAjaxSaveBlockContent()
             $blockPsr->custom_icon = '';
         }
         $blockPsr->date_add = date('Y-m-d H:i:s');
-        $blockPsr->date_update = date('Y-m-d H:i:s');
+        $blockPsr->date_upd = date('Y-m-d H:i:s');
```
Comment (matks) [inline controllers/admin/AdminBlockListingController.php:171]: Nice catch! So this PR not only improves the project, it introduces bug fixes too! 👏

File: upgrade/upgrade-4.0.0.php [hunk 2]
```diff
@@ -38,7 +38,7 @@
 function upgrade_module_4_0_0($module)
 {
     $tab = new Tab();
-    $tab->active = 1;
+    $tab->active = true;
```
Comment (matks) [inline upgrade/upgrade-4.0.0.php:41]: Did you check this works or do we need QA? I'm not 100% legacy module system accepts this

File: controllers/admin/AdminBlockListingController.php (hunk 1 above)
Comment (Progi1984) [reply controllers/admin/AdminBlockListingController.php:171]: :+1:

File: upgrade/upgrade-4.0.0.php (hunk 2 above)
Comment (Progi1984) [reply upgrade/upgrade-4.0.0.php:41]: I will make a release just after this PR. So yes a QA will be done.
I already fixed this bad assignment in the core with PHPStan (Level 2, i believe).

File: .github/workflows/php.yml [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    # Check the PHP code follow the coding standards
+    php-cs-fixer:
+        name: PHP-CS-Fixer
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v2.0.0
+
+            - name: Run PHP-CS-Fixer
+              uses: prestashopcorp/github-action-php-cs-fixer@master
```
Comment (PierreRambaud) [inline .github/workflows/php.yml:30]: Please don't use this one, it's not taking in consideration what you specify in your composer.lock

Comment (Progi1984) [reply .github/workflows/php.yml]: Fixed

**Review by NeOMakinG — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Pull request merged**